### PR TITLE
feat: add useRepayETH hook (#154)

### PR DESCRIPTION
**Pull request by aouahib**

### CODE REVIEW

**Comment by aouahib** [comment]
Current dependencies on/for this PR:
* main
  * **PR #153** <a href="https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/153" target="_blank">Graphite</a> 
    * **PR #155** <a href="https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/155" target="_blank">Graphite</a> 
      * **PR #156** <a href="https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/156" target="_blank">Graphite</a> 
        * **PR #157** <a href="https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/157" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/157?utm_source=stack-comment).

**Review by andrewszucs — APPROVED** [review]

**Comment by aouahib** [comment]
**Graphite Merge Job**

Current status: ⏳ Queued to merge

This pull request is currently queued to be merged as part of a stack.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/157?utm_source=stack-comment).


Job Reference: LhE62Pbrj0P1k6gpAemc

**Comment by aouahib** [comment]
**Graphite Merge Job**

Current status: 🔄 Waiting for checks to pass

This pull request is currently waiting for CI and mergeability checks to pass before being merged.

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/metastreet-labs/metastreet-sdk/157?utm_source=stack-comment).


Job Reference: W6it8zzsk3uVZvwrcgQc

**Pull request merged**
